controller/linodemachine_controller.go

```diff
@@ -460,7 +480,7 @@ func (r *LinodeMachineReconciler) resizeRootDisk(
 
 	// wait for the disk to resize
 	if _, err := machineScope.LinodeClient.WaitForInstanceDiskStatus(ctx, linodeInstanceID, rootDiskID, linodego.DiskReady, defaultResizeWaitSeconds); err != nil {
-		logger.Error(err, fmt.Sprintf("Failed to resize root disk within resize timeout of %d seconds", defaultResizeWaitSeconds))
+		logger.Info(fmt.Sprintf("Failed to resize root disk within resize timeout of %d seconds, requeuing", defaultResizeWaitSeconds))
```

Comment:
Don't put variable information into log messages. It hurts the cardinality when ingested into a logging system.

```suggestion
		logger.Info("Timed out resizing root disk",
			"timeout", defaultResizeWaitSeconds,
			"reqeue", true,
		)
```

Even though it would be a larger change, I would _strongly_ recommend using [time.Duration](https://pkg.go.dev/time#Duration) instead of doing a numeric value with the units embedded in the name.